scripts/port2alias

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import sys
+import subprocess
+from portconfig import get_port_config
+
+
+def get_platform_hwsku():
+    hwsku = None
+    platform = None
+    command = "show platform summary"
+    p = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE)
+    output = p.stdout.readlines()
+    for line in output:
+        tokens = line.split()
+        if not tokens:
+            continue
+        if tokens[0].lower() == 'hwsku:':
+            hwsku = tokens[1]
+        elif tokens[0].lower() == 'platform:':
+            platform = tokens[1]
+    return (platform, hwsku)
+
+def main():
+    (platform, hwsku) = get_platform_hwsku()
+    (port, _) = get_port_config(hwsku, platform)
+    for line in sys.stdin:
+        start = 0
+        end = 0
+        while end < len(line):
+            if line[end].isalnum():
+                pass
+            else: 
+                # End of a word
+                word = line[start:end]
+                if port.has_key(word):
```

Comment:
Consider `if word in port:` (or `if word in port.keys():` to make it more understandable at first glance) instead. More portable for the future as it appears has_key() is deprecated and has been removed in Python 3.